tests/data/l7irish/data.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import shutil
+from typing import Dict, List, Union
+
+import numpy as np
+import rasterio
+from rasterio import Affine
+from rasterio.crs import CRS
+
+SIZE = 36
+
+np.random.seed(0)
+
+FILENAME_HIERARCHY = Union[Dict[str, "FILENAME_HIERARCHY"], List[str]]
+
+bands = [
+    "B10.TIF",
+    "B20.TIF",
+    "B30.TIF",
+    "B40.TIF",
+    "B50.TIF",
+    "B61.TIF",
+    "B62.TIF",
+    "B70.TIF",
+    "B80.TIF",
+]
+
+filenames: FILENAME_HIERARCHY = {
+    "austral": {"p226_r98": [], "p227_r98": [], "p230_r95": []},
+    "boreal": {"p2_r27": [], "p128_r25": []},
```

Comment:
Some scenes are split into parts (p186_r32_1, p186_r32_2, etc.). Can you change one or two of these to match that so we know if can handle it properly?